.github/workflows/main.yml

```diff
@@ -28,10 +31,34 @@ jobs:
           python-version: "3.9"
       - uses: extractions/setup-just@v1
       - name: Run tests
-#       env:  # Add environment variables required for tests
         run: |
           just test
 
+  docker:
+    needs: [test]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - uses: extractions/setup-just@v1
+      - name: Build docker image and run tests in it
```

Comment:
Is this doubling up what the test action is doing?

Reply:
Yes, but it acts as a test that the docker image works. Which we have a bad habit of not doing.

And release-hatch tests are very fast, so it doesn't add much overhead.